api/test/integration/test_media_integration.py

```diff
@@ -329,6 +329,57 @@ def test_detail_view_for_invalid_uuids_returns_not_found(
     assert res.status_code == 404
 
 
+def test_search_with_only_valid_sources_produces_no_warning(media_type, api_client):
+    stats = api_client.get(f"/v1/{media_type.path}/stats/")
+    assert stats.status_code == 200
+
+    search = api_client.get(
+        f"/v1/{media_type.path}/",
+        {"source": ",".join([s["source_name"] for s in stats.json()])},
```

Comment:
We have some hardcoded provider names in the `media_type` fixture. By not making another request we can make the tests independent of the `/stats` endpoint.

Reply:
Done: https://github.com/WordPress/openverse/pull/4031/commits/caddd9eab6bce3dd81d8155872d62ee28e881163

Agree we should refactor that. Ideally we would unify those media type configurations with the ones that pre-existed them used in other tests too, rather than having two separate ones with slightly different names for things.